ui/public/mcp.js

```diff
@@ -4,8 +4,8 @@ console.log('Mobile Control Panel Extension Loaded');
 window.OPENSHIFT_CONSTANTS.PROJECT_NAVIGATION.splice(1, 0, {
```

Comment:
This file could probably use some formatting, indentation in some places is incorrect and there are some large gaps between lines